src/sentry/api/serializers/models/apitoken.py

```diff
@@ -27,6 +27,13 @@ def serialize(self, obj, attrs, user):
             "state": attrs.get("state"),
         }
         if not attrs["application"]:
-            data["token"] = obj.token
+            include_token = kwargs.get("include_token", False)
```

Comment:
When you split your PR note that this default setting should come last. Because if this merges before Frontend changes for a while user won't see the token at all.